client/src/Admin/Settings.js

```diff
@@ -1,12 +1,96 @@
-import React from 'react';
+import React, { useState, useContext } from 'react';
+
+import Context from '../Context';
+import ApiService from '../ApiService';
+import FormError from '../Models/FormError';
+import FormCheckbox from '../Components/FormCheckbox';
+
+import './Settings.scss';
+
+function SettingsEdit({ setShowEdit, contextOrganization, setContextOrganization }) {
+  const [organizationInEdit, setOrganizationInEdit] = useState(contextOrganization);
+  const [error, setError] = useState();
+
+  function onChange(property, value) {
+    const newOrg = { ...organizationInEdit };
+    newOrg[property] = value;
+    setOrganizationInEdit(newOrg);
+  }
+  function onSubmit() {
+    ApiService.organizations
+      .update(contextOrganization, organizationInEdit)
```

Comment:
Decided to patch with the whole organization object as opposed to just the value of the `isMfaEnabled` value in order to allow for more organization settings to be updated in the future. 
This is what I meant in the description of "Set the groundwork for showing and updating more organization settings"